test/manydbs/manydbs.c

```diff
@@ -205,16 +206,23 @@ main(int argc, char *argv[])
         testutil_check(get_stat(sessions[i], WT_STAT_CONN_COND_AUTO_WAIT_RESET, &cond_reset));
         testutil_check(get_stat(sessions[i], WT_STAT_CONN_COND_AUTO_WAIT, &cond_wait));
         /*
-         * On an idle workload there should be no resets of condition variables during the idle
-         * period. Even with a light workload, resets should not be very common. We look for 5%.
+         * On an idle workload there should be minimal resets of condition variables during the idle
+         * period, looking for 5%. Even with a light workload, resets should not be very common. We
+         * look for 10%.
```

Comment:
Did you take a look at the prototype patch suggested here:
https://github.com/sauclovian-wt/wiredtiger/commit/af5190ea808296b3fc4eaba642e5b146d3975735

It's an interesting approach because it allows us to remain strict in places where the test is already passing (Linux).

Reply:
I have now changed the approach to the patch. I have currently also added in checking NETBSD. let me know if I should remove it for now.